recipes/openal/all/conanfile.py

```diff
@@ -169,7 +169,7 @@ def package_info(self):
         self.cpp_info.includedirs.append(os.path.join("include", "AL"))
         if self.settings.os == "Linux":
             self.cpp_info.system_libs.extend(["dl", "m"])
-        elif self.settings.os == "Macos":
+        elif tools.is_apple_os(self.settings.os):
```

Comment:
Since today, `OpenALConfig.cmake` by openal-soft` also provides the cmake variables of `FindOpenAL.cmake` by CMake.
See https://github.com/kcat/openal-soft/pull/581
So you need to add the module file as a build module too for `cmake_find_package_multi`.

Reply:
You mean that I should add the file conan-official-openal-variables.cmake to cmake_find_package_multi also? If yes, the content should be modified because `OpenAL_INCLUDE_DIR` etc don't exist in a config file.

Reply:
All fine to me :)
I didn't know about `OpenAL_INCLUDE_DIR`